operator/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 ## Main
 
+## Release 5.6.21
+
+- [13369](https://github.com/grafana/loki/pull/13369) **jatinsu**: feat(operator): Add support for the volume API
+
 ## Release 5.6.20
 
+No changes.
+
```

Comment:
Seems like a typo or do we not consider Robert's PR a noteworthy change?

Reply:
Which PR?